src/main/java/io/vertx/core/loadbalancing/LoadBalancer.java

```diff
@@ -64,6 +69,66 @@ public <E> Endpoint<E> endpointOf(E endpoint) {
     }
   };
 
+  /**
+   * Random load balancer.
+   */
+  // ThreadLocalRandom? contention, seed?
+  LoadBalancer RANDOM = () -> {
+    Random random = new Random();
+    return (EndpointSelector) endpoints -> {
+      if (endpoints.isEmpty()) {
+        return -1;
+      }
+      return random.nextInt(endpoints.size());
+    };
+  };
+
+  /**
+   * Power of two choices load balancer.
+   */
+  // BEST_OF_TWO? P2C?
+  LoadBalancer POWER_OF_TWO_CHOICES = () -> new EndpointSelector() {
```

Comment:
wdyt about the naming?

Reply:
Power of two choices is the generally accepted name, chosen after this paper: https://www.nginx.com/blog/nginx-power-of-two-choices-load-balancing-algorithm/

Reply:
(okk, just in case it was long for you)